CHANGELOG.md

```diff
@@ -375,3 +375,10 @@ All notable changes to this project will be documented in this file. The format
 ### Fixed
  - Fix: AKO doesn’t support Service of type LoadBalancer definitions with mixed protocols.
  - Fix: CTRL_CA_DATA env variable, from AKO Statefulset definition, violates CNTR-K8-001160.
+
+## AKO-1.10.3
```

Comment:
should we add the new field `networkUUID` in `values.yaml` under `Added`?

Reply:
Addressed